src/streamlit_extras/card/__init__.py

```diff
@@ -0,0 +1,20 @@
+from streamlit_card import card
+from .. import extra
+
+card = extra(card)
+
+def example():
+    card(
+        title="Hello World!",
+        text="Some description",
+        image="http://placekitten.com/300/250"
+    )
+
+__title__ = "Streamlit Card"  # title of your extra!
+__desc__ = "Streamlit Component, for a UI card"  # description of your extra!
+__icon__ = "🖼️"  # give your extra an icon!

```

Comment:
```suggestion
__icon__ = "💳"  # give your extra an icon!
```

If you don't mind, that 🖼️ is taken for the upcoming [Chart container extra](https://github.com/arnaudmiribel/streamlit-extras/pull/80) so suggested another one 💳 (feel free to use another). I know, some day will come where emojis will inevitably collide given the amount of extras we get.. but until then...